verify-merge.py

```diff
@@ -3,41 +3,54 @@
 import os
 import subprocess
 import glob
+import sys
 
 GIT = os.getenv('GIT','git')
 GPG = os.getenv('GPG','gpg')
 
 def verify():
     global args, workdir
-    os.chdir('gitian-pubkeys')
-    print('Importing pubkeys...')
-    keys = [f for f in glob.glob("*.asc", recursive=True)]
-    for key in keys:
-        subprocess.check_call([GPG, '--import', key])
-    print('Refreshing pubkeys...')
-    subprocess.check_call([GPG, '--refresh'])
-    os.chdir('../../gitian-builder')
-    print('\nVerifying '+args.version+' Linux\n')
-    subprocess.check_call(['bin/gverify', '-v', '-d', '../gitian.sigs/', '-r', args.version+'-linux', '../monero/contrib/gitian/gitian-linux.yml'])
-    print('\nVerifying '+args.version+' Windows\n')
-    subprocess.check_call(['bin/gverify', '-v', '-d', '../gitian.sigs/', '-r', args.version+'-win', '../monero/contrib/gitian/gitian-win.yml'])
-    print('\nVerifying '+args.version+' MacOS\n')
-    subprocess.check_call(['bin/gverify', '-v', '-d', '../gitian.sigs/', '-r', args.version+'-osx', '../monero/contrib/gitian/gitian-osx.yml'])
+    if args.import_keys:
+        os.chdir('gitian-pubkeys')
+        print('Importing pubkeys...')
+        keys = [f for f in glob.glob("*.asc", recursive=True)]
+        for key in keys:
+            subprocess.check_call([GPG, '--import', key])
+        os.chdir('../')
+    if args.refresh_keys:
+        print('Refreshing pubkeys...')
+        subprocess.check_call([GPG, '--refresh'])
+    if not os.path.isdir(args.gitian_builder_dir):
+        sys.stderr.write('Please clone the gitian-builder repository from github.com/devrandom/gitian-builder to the directory containing the gitian.sigs repository.\n If you already have the gitian.sigs directory cloned, but under another name or path, use --gitian-builder-dir to pass its absolute directory path to the script.')
```

Comment:
Nit: would be nice to end with a `\n` so that the prompt returns to a new line after an error and remove the space before "If" so the line isn't indented.